.github/workflows/build-against-langchain4j.yml

```diff
@@ -35,14 +35,16 @@ jobs:
 
       - name: Build langchain4j
         run: |
-          mkdir langchain4j
-          cd langchain4j
           git clone https://github.com/langchain4j/langchain4j.git
           cd langchain4j
           ./mvnw -B clean install -DskipTests -DskipITs
 
       - name: Build with Maven
-        run: ./mvnw -B clean install -Dno-format
+        run: |
+          sudo apt-get install libxml2-utils
+          LANGCHAIN4J_VERSION=$(xmllint --xpath '/*[local-name()="project"]/*[local-name()="version"]/text()' langchain4j/pom.xml)
```

Comment:
`LANGCHAIN4J_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)`

seems much simpler to me :)